### PR TITLE
assertions checkDesign

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Review by sachseka — COMMENTED** [review]
LGTM! But please pay attention to checks (edit testthat), if not tests will fail

**Review by sachseka — CHANGES_REQUESTED** [review]
Please pay attention to checks (in this case testthat tests)

**Comment by grewered** [comment]
error: data frames are type character - see https://github.com/sachseka/eatPrep/pull/25

File: R/checkDesign.r [hunk 1]
```diff
@@ -1,4 +1,10 @@
 checkDesign <- function(dat, booklets, blocks, rotation, sysMis = "NA", id = "ID", subunits = NULL, verbose = TRUE) {
+  checkmate::assert_data_frame(dat)
+  lapply(c(booklets, blocks, rotation), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/checkDesign.r:3]: please use list(booklets, blocks, rotation)

**Review by sachseka — CHANGES_REQUESTED** [review]

**Review by sachseka — APPROVED** [review]

**Pull request merged**